vignettes/long-format.Rmd

```diff
@@ -9,51 +9,124 @@ vignette: >
 
 ```{r, include = FALSE}
 knitr::opts_chunk$set(
-  collapse = TRUE,
-  comment = "#>"
+  collapse  = TRUE,
+  comment   = "#>",
+  out.width = "100%"
 )
 ```
 
+
+
 ```{r setup}
 library(funbiogeo)
 ```
 
-If your data are not split into three tables you can use the functions
-`fb_format_*()` to create these specific objects. These functions take a long
-table as input, where one row corresponds to the occurrence/abundance/coverage 
-f one species at one site and output a tidy object. Similarly, if your data is
-in a long format some variables are repeated across the table
-(e.g. sites coordinates, species traits).
 
-For example, the raw data used to generate the internal data of `funbiogeo`:
+The `funbiogeo`package requires that information is structured in three different datasets:
+
+- the **species x traits** `data.frame` (`species_traits` in `funbiogeo`)
+- the **site x species** `data.frame` (`site_species` in `funbiogeo`)
+- the **site x locations** object (`site_locations` in `funbiogeo`)
+
+
+
```

Comment:
Just a question: you always seem to add so much space between paragraphs. You don't like having single empty lines or you prefer to have more space in the final document?
(Real question not a suggestion😅)

Reply:
Yes 😄  I like that my code breathe 😄 othewise I suffocate 😄